catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -245,8 +245,8 @@ def __init__(self, dm_config_id, wavelength, testbed_config_id, apply_flat_map=F
             self.segments_in_pupil = util.iris_pupil_naming(self.dm_config_id)
 
         # Initalize parameters for displaying the command
-        self.instrument_fov = CONFIG_INI.getint(testbed_config_id, 'fov')
-        self.pixelscale = CONFIG_INI.getfloat(testbed_config_id, 'pixelscale')
+        self.instrument_fov = CONFIG_INI.getint(testbed_config_id, 'fov_irisao_plotting')
+        self.pixelscale = CONFIG_INI.getfloat(testbed_config_id, 'pixelscale_iriso_plotting')
```

Comment:
Typo: ``iriso`` -> ``risao``, however, it was also missing in https://github.com/spacetelescope/hicat-package/pull/399 so I'm just going to merge as is.